buildSrc/src/main/kotlin/com.hedera.block.jpms-modules.gradle.kts

```diff
@@ -117,4 +146,22 @@ extraJavaModuleInfo {
     module("com.google.auto.service:auto-service-annotations", "com.google.auto.service")
     module("com.google.auto.service:auto-service", "com.google.auto.service.processor")
     module("com.google.auto:auto-common", "com.google.auto.common")
+
+    // JMH only
+    module("net.sf.jopt-simple:jopt-simple", "jopt.simple")
+    module("org.openjdk.jmh:jmh-core", "jmh.core")
+    module("org.openjdk.jmh:jmh-generator-asm", "jmh.generator.asm")
+    module("org.openjdk.jmh:jmh-generator-bytecode", "jmh.generator.bytecode")
+    module("org.openjdk.jmh:jmh-generator-reflection", "jmh.generator.reflection")
+
+    module("com.google.protobuf:protobuf-java-util", "com.google.protobuf.util")
+    module("junit:junit", "junit")
+    module("org.hamcrest:hamcrest", "org.hamcrest")
+    module("org.json:json", "org.json")
+    module("org.mockito:mockito-core", "org.mockito")
+    module("org.objenesis:objenesis", "org.objenesis")
+    module("org.rnorth.duct-tape:duct-tape", "org.rnorth.ducttape")
+    module("org.testcontainers:junit-jupiter", "org.testcontainers.junit.jupiter")
+    module("org.testcontainers:testcontainers", "org.testcontainers")
+    module("org.mockito:mockito-junit-jupiter", "org.mockito.junit.jupiter")
 }
```

Comment:
We need to add the following at the end of this file:

```
// Make 'javax.annotation:javax.annotation-api' discoverable for merging it into
// 'com.google.code.findbugs:jsr305'
dependencies { "javaModulesMergeJars"("javax.annotation:javax.annotation-api:1.3.2") }
```

It's working at the moment because of the `dependencies { }` blocks in `build.gradle.kts` files, but that's not the "right" way to fix it. On the long run, I hope we can get rid of the "Jar merge". Right now we have the same issue in all repositories. This is (more ore less) consistent with what we do here for example: https://github.com/hashgraph/hedera-sdk-java/blob/main/gradle/plugins/src/main/kotlin/com.hedera.gradle.patch-modules.gradle.kts#L165 (although there we do it the other way around).

I will align this in all repos and re-evaluate what the best common solution is as part of https://github.com/hashgraph/hedera-services/issues/14255.

Reply:
I added this to the end of the jpms plugin. Thanks for the fix